src/Build/Evaluation/Expander.cs

```diff
@@ -2810,6 +2811,46 @@ internal static ItemTransformFunction GetItemTransformFunction(IElementLocation
                     }
                 }
 
+                /// <summary>
+                /// Intrinsic function that returns those items don't have the given metadata value
+                /// Using a case insensitive comparison.
+                /// </summary>
+                /// 
+                internal static IEnumerable<Pair<string, S>> WithOutMetadataValue(Expander<P, I> expander, IElementLocation elementLocation, bool includeNullEntries, string functionName, IEnumerable<Pair<string, S>> itemsOfType, string[] arguments)
```

Comment:
nit:
```suggestion
                internal static IEnumerable<Pair<string, S>> WithoutMetadataValue(Expander<P, I> expander, IElementLocation elementLocation, bool includeNullEntries, string functionName, IEnumerable<Pair<string, S>> itemsOfType, string[] arguments)
```

(Without is one word)

Reply:
Thanks. Have updated